tests/50federation/01keys.pl

```diff
@@ -65,7 +62,7 @@
 
          log_if_fail "Signed bytes", $signed_bytes ;
 
-         $crypto_sign->verify( $signature, $signed_bytes, $key ) or
+         Crypt::Ed25519::verify( $signed_bytes, $key, $signature ) or
```

Comment:
Do we need to assert the length of the signature, like we did in `verify_json_signature`?

(This replacement seems to be done in a few places -- if the answer is yes, should we have a helper to do this?)

Reply:
hum, I guess we should, otherwise there is nothing that checks if there is garbage after the signature.

Reply:
I chose to make the check explicit rather than add a helper, since it's not too long and I think it's clearer.